DotnetCLIVersion.txt

```diff
@@ -0,0 +1 @@
+1.0.0-beta-002228
```

Comment:
Doesn't have to be with this PR. This looks fine. The general format is:

`https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/{version}/dotnet-dev-{os}-{arch}-{version}.{zip|tar.gz}`

`https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/1.0.0-rc2-002416/dotnet-dev-osx-x64.1.0.0-rc2-002416.tar.gz`

`https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/1.0.0-rc2-002416/dotnet-dev-ubuntu-x64.1.0.0-rc2-002416.tar.gz`

`https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/1.0.0-rc2-002416/dotnet-dev-win-x64.1.0.0-rc2-002416.zip`